### PR TITLE
feat: added data-css to essential html tags

### PR DESCRIPTION
**Pull request by jwasiak**

### CODE REVIEW

File: src/frontend/components/actions/list.tsx [hunk 1]
```diff
@@ -54,8 +55,10 @@ const List: React.FC<ActionProps> = ({ resource, setTag }) => {
     navigate({ search: search.toString() })
   }
 
+  const contentTag = getResourceElementCss(resource, 'list-content')
```
Comment (dziraf) [inline src/frontend/components/actions/list.tsx:58]: Why not `getActionElementCss`? This is a `list` action component.

The element name could be `table-wrapper`

File: src/frontend/components/app/action-header/action-header.tsx [hunk 2]
```diff
@@ -84,9 +84,9 @@ const ActionHeader: React.FC<ActionHeaderProps> = (props) => {
   const cssHeaderMT = action.showInDrawer ? '' : 'lg'
   const cssActionsMB = action.showInDrawer ? 'xl' : 'default'
   const CssHComponent = action.showInDrawer ? H3 : H2
-
+  const contentTag = `${resourceId}-${action.name}-action-header`
```
Comment (dziraf) [inline src/frontend/components/app/action-header/action-header.tsx:87]: `getActionElementCss`?

File: src/frontend/components/app/action-header/actions-to-button-group.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export const actionsToButtonGroup = (
       onClick: href ? handleClick : undefined,
       'data-testid': buildActionTestId(action),
       buttons: [],
+      'data-css': `${action.resourceId}-${action.name}-button`,
```
Comment (dziraf) [inline src/frontend/components/app/action-header/actions-to-button-group.ts:29]: `getActionElementCss`?

File: src/frontend/components/app/action-button/action-button.tsx [hunk 4]
```diff
@@ -59,9 +59,12 @@ const ActionButton: React.FC<ActionButtonProps> = (props) => {
     throw new Error('ActionButton has to have one child')
   }
 
+  const contentTag = `${resourceId}-${action.name}-button`
```
Comment (dziraf) [inline src/frontend/components/app/action-button/action-button.tsx:62]: `getActionElementCss`?

File: src/frontend/components/app/breadcrumbs.tsx [hunk 5]
```diff
@@ -81,9 +81,9 @@ const Breadcrumbs: React.FC<BreadcrumbProps> = (props) => {
   const action = resource.actions.find((a) => a.name === actionName)
   const h = new ViewHelpers()
   const { translateLabel: tl } = useTranslation()
-
+  const contentTag = `${resource.id}-${actionName}-breadcrumbs`
```
Comment (dziraf) [inline src/frontend/components/app/breadcrumbs.tsx:84]: `getActionElementCss`?

File: src/frontend/components/property-type/base-property-component.tsx [hunk 6]
```diff
@@ -69,6 +69,7 @@ const BasePropertyComponent: React.FC<BasePropertyComponentProps> = (props) => {
   }), [baseProperty])
 
   const testId = `property-${where}-${property.path}`
+  const contentTag = `${resource.id}-${where}-${property.path}`
```
Comment (dziraf) [inline src/frontend/components/property-type/base-property-component.tsx:72]: `getActionElementCss`? `where` is just an action name

**Review by dziraf — CHANGES_REQUESTED** [review]

File: src/frontend/components/app/action-header/action-header.tsx (hunk 2 above)
Comment (jwasiak) [reply src/frontend/components/app/action-header/action-header.tsx]: done

File: src/frontend/components/actions/list.tsx (hunk 1 above)
Comment (jwasiak) [reply src/frontend/components/actions/list.tsx]: done

File: src/frontend/components/app/action-header/actions-to-button-group.ts (hunk 3 above)
Comment (jwasiak) [reply src/frontend/components/app/action-header/actions-to-button-group.ts:29]: done

File: src/frontend/components/app/action-button/action-button.tsx (hunk 4 above)
Comment (jwasiak) [reply src/frontend/components/app/action-button/action-button.tsx]: done

File: src/frontend/components/property-type/base-property-component.tsx (hunk 6 above)
Comment (jwasiak) [reply src/frontend/components/property-type/base-property-component.tsx]: done

**Pull request merged**